### PR TITLE
UHM-5750 - dates only show if they user selected the checkbox and the…

### PR DESCRIPTION
**Pull request by louidorjp**
Please Review this PR
@lnball

### CODE REVIEW

File: configuration/pih/scripts/global/zl.js [hunk 1]
```diff
@@ -87,3 +87,33 @@ function setUpExpandableContacts(badPhoneNumberMsg) {
   }
 
 }
+
+function showDateIfCheckboxSelected(){
```
Comment (brandones) [inline configuration/pih/scripts/global/zl.js:91]: This looks like what the `toggle` attribute of the [obs tag](https://wiki.openmrs.org/display/docs/HTML+Form+Entry+Module+Reference#HTMLFormEntryModuleReference-%3Cobs%3E) is supposed to do.

Comment (louidorjp) [reply configuration/pih/scripts/global/zl.js:91]: @lnball @brandones 
Of course, but it didn't work for the checkbox  or maybe the answerConceptId doesn't match. that's why I created this function.

File: configuration/pih/htmlforms/hiv/section-hiv-state.xml [hunk 2]
```diff
@@ -922,6 +922,73 @@
...
     </section>
 
+    <!-- this section for PMTCT followup -->
+    <includeIf velocityTest="$encounter.encounterType.uuid == '95e03e7d-9aeb-4a99-bd7a-94e8591ec2c5'">
+        <section id="syphilis-section" sectionTag="fieldset"
+                 headerStyle="title" headerCode="Syphilis">
+
+            <div class="section-container">
+                <p>
+                    <obs id="syph-q"  conceptId="CIEL:164801" style="checkbox"
```
Comment (brandones) [inline configuration/pih/htmlforms/hiv/section-hiv-state.xml:932]: Get rid of `id`s that aren't used

```suggestion
                    <obs  conceptId="CIEL:164801" style="checkbox"
```

File: configuration/pih/htmlforms/hiv/section-hiv-state.xml [hunk 3]
```diff
@@ -922,6 +922,73 @@
...
+                            <p>
+                                <obs id="mom-syph" conceptId="CIEL:1628" style="checkbox"
+                                     answerConceptId="CIEL:112493"
+                                     answerCode="pihcore.syphilisTreatmentPregnantMother" toggle="mom-syph-treat"/>
+                            </p>
+                            <p id="mom-syph-treat">
+                                <label>
+                                    <uimessage code="pihcore.treatmentStartDate" />
+                                </label>
+                                <obs conceptId="CIEL:163526"  />
```
Comment (brandones) [inline configuration/pih/htmlforms/hiv/section-hiv-state.xml:958]: Any reason not to do this:

```suggestion
                                <obs conceptId="CIEL:163526" labelCode="pihcore.treatmentStartDate" />
```

?

File: configuration/pih/htmlforms/hiv/section-pmtct-history.xml [hunk 4]
```diff
@@ -456,13 +456,9 @@
...
                      answerConceptId="CIEL:1065" answerCode="pihcore.screeningSyph"
-                     dateLabel="Date" >
-                    <controls>
-                        <when value="CIEL:1065" thenDisplay="#syph-screen"/>
-                    </controls>
-                </obs>
+                     dateLabel="Date" toggle="syph-screen"/>
             </p>
-            <p id="syph-screen" class="side-by-side">
+            <p id="syph-screen" class="side-by-side" style="display:none">
```
Comment (brandones) [inline configuration/pih/htmlforms/hiv/section-pmtct-history.xml:461]: This is confusing to me, but if you tested it it's probably fine.

File: configuration/pih/htmlforms/hiv/section-pmtct-history.xml [hunk 5]
```diff
@@ -499,9 +492,12 @@
 
                 <div>
                     <obsgroup groupingConceptId="CIEL:166558">
-                        <obs conceptId="CIEL:1628" style="checkbox"
+                        <p>
+                        <obs  id="partner-syph" conceptId="CIEL:1628" style="checkbox"
```
Comment (brandones) [inline configuration/pih/htmlforms/hiv/section-pmtct-history.xml:496]: Please fix indentation

**Comment by brandones** [comment]
Nice work, thanks @louidorjp !

**Pull request merged**